packages/client/src/pages/admin_preferences/index.tsx

```diff
@@ -98,7 +98,7 @@ const OrganizationSettings: React.FC<Props> = () => {
               <Divider />
               <Form className={classes.form}>
                 <FormControl component="fieldset">
-                  <FormSection content={nameField} />
+                  <FormSection content={publicInfoFields} name="Public Info" />
```

Comment:
I wouldn't name this "Public info" but rather just "General". Since admin using our app doesn't need to care not know which data will be available to whom (in this sense). It's up to the app to make sure that appropriate data is available to appropriate parties, without bothering the user with the implementation. 
Additionally there might be some "General" data which won't be available to the public. At which point, the distinction would be made on the implementation level (as I've described with the data trigger logic).

Also, this should be translateble, so it's best to provide the translation logic right away and simply add the italian translation later.